advanced_functionality/pytorch_yolov5_training_and_hpo/transfer_learning_and_hpo_yolov5.ipynb

```diff
@@ -0,0 +1,1379 @@
+{
```

Comment:
<p>Break up sentence -&gt; "file that has to follow a specific structure. In this structure "</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3601/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='75'/>

Reply:
<p>Broke up the sentence, this is what it looks like now, let me know if this is OK:</p><p> </p><p>Whether you run a training job, or perform automated model tuning (AMT), you need to create a <code>model.tar.gz</code> file that has to follow a specific structure. In this structure you will need to have the model, in this case the <code>best.pt</code> file. This is available in the <code>weights</code> directory in your <code>model.tar.gz</code> file. For a deeper understanding of how to create the model directory structure refer to <a href="https://sagemaker.readthedocs.io/en/stable/frameworks/pytorch/using_pytorch.html#deploy-pytorch-models" target="_blank">Use PyTorch with SageMaker SDK</a>.</p>